OsStub/BaseCryptLibOpenssl/Hash/CryptSm3.c

```diff
@@ -209,26 +209,28 @@ Sm3HashAll (
   OUT  UINT8       *HashValue
   )
 {
-  SM3_CTX Ctx;
+  INTN success;
+  EVP_MD_CTX* ctx;
 
   //
   // Check input parameters.
   //
-  if (HashValue == NULL) {
+  if (!HashValue)

```

Comment:
This is not acceptable.
HashValue is *UINT8, not boolean.

Please change it back.